.github/workflows/pr-label-on-approved.yml

```diff
@@ -1,5 +1,5 @@
-on: pull_request_review
-name: Label approved pull requests
+name: "Label approved pull requests"
+on: pull_request_target
 
 permissions:
   contents: read          # Required for checking changed files
```

Comment:
Not directly related to this PR, but I don't think this permission is needed in this workflow?

Reply:
Those permissions are not needed, true (i guess copy / paste left over) - otherwise I don't know how to fix the problem of this PR.